btc_parser/sources/utils.move

```diff
@@ -0,0 +1,139 @@
+module btc_parser::utils;
+use std::u64::do;
+use std::hash::sha2_256;
+
+
+#[test_only]
+use std::unit_test::assert_eq;
+
+#[error]
+const EOutOfBounds: vector<u8> = b"Slice out of bounds";
+
+/// covert vector bytes in little endian to number
```

Comment:
misspelling, use capital letters, lets make the comments consitent. For the naming I prefer `to_u64` this is what we use in other methods eg. `u64_to_varint_bytes`. There is no such type as `number` in move so i think it makes more sense